boards/arm/v2m_musca/doc/v2m_musca.rst

```diff
@@ -0,0 +1,412 @@
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | watchdog                            |
++-----------+------------+-------------------------------------+
+| TIMER     | on-chip    | timer                               |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by the port.
+See the `V2M Musca Website`_ for a complete list of V2M Musca board hardware
+features.
+
+The default configuration can be found in the defconfig file:
+
+.. code-block:: console
+
+   boards/arm/v2m_musca/v2m_musca_defconfig
+
+Interrupt Controller
+====================
+
+Musca is a Cortex-M33 based SoC and has 15 fixed exceptions and 77 IRQs.
+
+A Cortex-M33-based board uses vectored exceptions. This means each exception
+calls a handler directly from the vector table.
+
+Handlers are provided for exceptions 1-7, 11-12, and 14-15. The table here
+identifies the handlers used for each exception.
+
++------+------------+----------------+--------------------------+
+| Exc# | Name       | Remarks        | Used by Zephyr Kernel    |
++======+============+================+==========================+
+| 1    | Reset      |                | system initialization    |
++------+------------+----------------+--------------------------+
+| 2    | NMI        |                | system fatal error       |
++------+------------+----------------+--------------------------+
+| 3    | Hard fault |                | system fatal error       |
++------+------------+----------------+--------------------------+
+| 4    | MemManage  | MPU fault      | system fatal error       |
++------+------------+----------------+--------------------------+
+| 5    | Bus        |                | system fatal error       |
++------+------------+----------------+--------------------------+
+| 6    | Usage      | undefined      | system fatal error       |
+|      | fault      | instruction,   |                          |
+|      |            | or switch      |                          |
+|      |            | attempt to ARM |                          |
+|      |            | mode           |                          |
++------+------------+----------------+--------------------------+
+|  7   |SecureFault | Unauthorised   |    system fatal error    |
+|      |            | access to      |                          |
+|      |            | secure region  |                          |
+|      |            | from ns space  |                          |
++------+------------+----------------+--------------------------+
```

Comment:
Just for completeness, I'd add the rows for exceptions 8-10 and 13 with the exception name, and indicate in the remarks column that it's not handled by Zephyr (maybe ``Not handled``)

Reply:
I don't think this table should even be here.

Reply:
Should I remove the whole Interrupt Controller section including this table?

Reply:
Yes

Reply:
> I don't think this table should even be here.
> Should I remove the whole Interrupt Controller section including this table?

Note that we have  a similar case in #8408 where I also suggested to remove the whole "Interrupt Controller" section (which was  "TBD" there). We should give users detailed instruction on board bring-up with Zephyr (i.e. building/flashing), and just link to manuals for peculiar info like IRQ maps.